docs/api/liquidstaking.md

```diff
@@ -0,0 +1,476 @@
+  "fee_pool_address": "canto1sdl4z9y8x59979qjx8ut9zyndsux9sld0s6kcv"
+}
+```
+
+## WithdrawInsuranceRequests
+
+Query WithdrawInsuranceRequests.
+
+Example Request
+
+```bash
+http://localhost:1317/canto/liquidstaking/v1/withdraw_insurance_requests
+```
+
+Example Response
+
+```json
+{
+  "withdraw_insurance_requests": [
+    {
+      "insurance_id": "7"
+    }
+  ],
+  "pagination": {
+    "next_key": null,
+    "total": "1"
+  }
+}
+```
+
+## WithdrawInsuranceRequest
+
+Query a WithdrawInsuranceRequest by insurance id.
+
+Example Request
+
+```bash
+http://localhost:1317/canto/liquidstaking/v1/insurances/5/withdraw_insurance_requests
+```
+
+Example Response
+
+```json
+{
+  "withdraw_insurance_request": {
+    "insurance_id": "5"
+  }
+}
+```
+
+## UnpairingForUnstakingChunkInfos
+
+Query UnpairingForUnstakingChunkInfos.
+
+Example Request
+
+```bash
+http://localhost:1317/canto/liquidstaking/v1/unpairing_for_unstaking_chunk_infos 
+```
+
+Example Response
+
+```json
+{
+  "unpairing_for_unstaking_chunk_info": {
+    "chunk_id": "2",
+    "delegator_address": "canto1xjlslz2vl7v6gu807fmfw8ae7726q9pf9t3x34",
+    "escrowed_lstokens": {
+      "denom": "lscanto",
+      "amount": "240214408039107442750000"
+    }
+  }
+}
+```
+
+## UnpairingForUnstakingChunkInfo
+
+Query an UnpairingForUnstakingChunkInfo by chunk id.
+
+Example Request
+
+```bash
+http://localhost:1317/canto/liquidstaking/v1/chunks/2/unpairing_for_unstaking_chunk_infos
+```
+
+Example Response
+
+```json
+{
+  "unpairing_for_unstaking_chunk_info": {
+    "chunk_id": "2",
+    "delegator_address": "canto1xjlslz2vl7v6gu807fmfw8ae7726q9pf9t3x34",
+    "escrowed_lstokens": {
+      "denom": "lscanto",
+      "amount": "240214408039107442750000"
+    }
+  }
+}
+```
+
+## RedelegationInfos
+
+Query RedelegationInfos.
+
+Example Request
+
+```bash
+http://localhost:1317/canto/liquidstaking/v1/redelegation_infos
+```
+
+Example Response
+
+```json
+{
+  "redelegation_infos": [
+    {
+      "chunk_id": "1",
+      "completion_time": "2030-09-08T06:25:48.694135Z"
+    }
+  ],
+  "pagination": {
+    "next_key": null,
+    "total": "1"
+  }
+}
+```
+
+## RedelegationInfo
+
+Query RedelegationInfo by chunk id.
+
+Example Request
+
+```bash
+http://localhost:1317/canto/liquidstaking/v1/chunks/redelegation_infos
+```
+
+Example Response
+
+```json
+{
+  "redelegation_info": {
+    "chunk_id": "1",
+    "completion_time": "2030-09-08T06:25:48.694135Z"
+  }
+}
+```
+
+## ChunkSize
+
+Query ChunkSize.
+
+Usage
+
+```bash
+chunk-size
+```
+
+Example
+
+```bash
+# Query chunk size
+cantod query liquidstaking chunk-size -o json | jq
+```
+
+## MinimumCollateral
+
+Query MinimumCollateral.
+
+Usage
+
+```bash
+minimum-collateral
+```
+
+Example
+
+```bash
+# Query minimum collateral  
+cantod query liquidstaking minimum-collateral -o json | jq
+```
+
+## States
+
+Query net amount state.
+
+Usage
+
+```bash
+states
+```
+
+Example
+
+```bash
+cantod query liquidstaking states -o json | jq
+```
```

Comment:
Why not switch to 'example request' and 'example request' to be consistent with the other examples?

Reply:
Alternatively, I suppose you could add that part to the CLI command.

Reply:
https://github.com/b-harvest/Canto/blob/zsystm/docs-add-and-updates/docs/cli/liquidstaking.md?plain=1#L190
In the Transaction section, some have the surfix "Cmd" in the command name and some don't. It would be nice to unify them.

Reply:
Applied it in [this commit](https://github.com/b-harvest/Canto/pull/15/commits/14beb18a4543dc2e64b82b42e1be43b58daa201b).
Thanks!